frontend/src/Views/MainPage/MainPage.tsx

```diff
@@ -38,7 +38,7 @@ const MainPage: React.FC = () => {
 
   return (
     <StyledMain>
-      <Header />
+      <MainPageHeader />
```

Comment:
Nice! It looks a bit weird now, but when we improve the form page, having this second header is going to be good for UI/UX

Reply:
What do you mean with it looks weird now?

Reply:
The main page is a bit naked, once we have the status bar, the footer and more modules, the center shift and resize of the header will make more sense. I got a bit weirded at first because the header was shifting, it didn't deserve a comment in retrospect. 

All I wish I'd saied was Good job with the main page Header.

After I tested the status bar though, I also noted the Header is scrolling away when the form gets large enough to enable scrolling:

![image](https://github.com/bcgov/nr-sustainment-capstone-2024/assets/80939533/174fa89d-b16a-4389-865a-0a86193a6fe2)

This might be worth taking a look at.

Anyways, thanks for your contribution, the header on the main page is looking nicer :tada: